health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -396,6 +417,9 @@ export const UICustomizations = {
       const [showPopup, setShowPopup] = useState(false);
 
       switch (key) {
+        case "MICROPLAN_FACILITY_ASSIGNED_VILLAGES":
+          const assignedVillages = row?.additionalDetails?.assignedVillages; 
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)